lxd/images.go

```diff
@@ -416,9 +416,24 @@ func imgPostRemoteInfo(d *Daemon, r *http.Request, req api.ImagesPost, op *opera
 		info.Properties[k] = v
 	}
 
+	// Get profile IDs
+	if req.Profiles == nil {
+		req.Profiles = []string{"default"}
```

Comment:
Should use `project.Default` instead of the hardcoded string.

Reply:
Fixed.